### PR TITLE
REL: Bump versioning to RocketPy v1.3.0

### PR DESCRIPTION
**Pull request by phmbressan**
## Pull request type

- [X] Code changes (bugfix, features)
- [X] Other (please describe): Release PR

## Checklist

- [X] Lint (`black rocketpy/ tests/`) has passed locally 
- [ ] All tests (`pytest tests -m slow --runslow`) have passed locally
- [X] `CHANGELOG.md` has been updated (if relevant)

## Current behavior

RocketPy version was v1.2.2.

## New behavior

This PR prepares the release of RocketPy v1.3.0. Featuring MonteCarlo simulations, code speed improvements and many other features and bug fixes.

## Breaking change

- [ ] Yes
- [X] No

### CODE REVIEW

**Review by Gui-FernandesBR — APPROVED** [review]
Amazing work! These 750 commits demonstrate the bright of our Team. 

Thank you everyone for the effort!

File: CHANGELOG.md [hunk 1]
```diff
@@ -25,10 +25,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 -->
 
-## [Unreleased] - yyyy-mm-dd
+## [1.3.0] - 2024-06-01
```
[the remark this answers is not in the document]
Comment (Gui-FernandesBR) [reply CHANGELOG.md:29]: Btw, before I forget ....

Could you actually add you PR (#614) to the CHANGELOG? 🤣🤣

[the remark this answers is not in the document]
Comment (phmbressan) [reply CHANGELOG.md]: Both done on [ecfe5f4](https://github.com/RocketPy-Team/RocketPy/pull/614/commits/ecfe5f4a439ed8ba9e12cd4d37323a4cd3ac8e86).

**Review by Gui-FernandesBR — APPROVED** [review]
LGTM

**Review by MateusStano — APPROVED** [review]

**Pull request merged**